src/server.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
 import { extractCritical } from 'emotion-server';
 import { renderHtmlDocument } from './utils/renderHtmlDocument';
-import { fetchDefaultEpicContent, fetchConfiguredEpicTests } from './api/contributionsApi';
```

Comment:
Is there any reason to keep `fetchDefaultEpicContent` around after this has been merged? Will we still need the default to fall back on in some cases?

Reply:
I think Frontend does have the default Epic hardcoded in the app somewhere, might be worth digging to see exactly when it's used there.

Reply:
I've dug around a bit in frontend and I can't find a place where we fall back to a default epic component. There's [some logic in `contributions-utilities.js`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js#L439) which  falls back to some default epic copy if the configured test doesn't have any copy, but from what I can tell it looks unlikely that we'd get to the point of using the default copy given the way we process the configured epic tests.

There's also a scenario downstream of [`displayControlEpic` in `epic-utils.js`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/epic/epic-utils.js#L92) where we fetch the default epic content. But the only place `displayControlEpic` is called is from `displayControlEpicInAbTest` and that's not used anywhere.

So my feeling is that I think we're safe to proceed without the fallback for now. Interested in other's thoughts. If anyone is interested to double check my investigation that'd make me feel more confident!